[{"client_msg_id":"ac2a9e0d-fdf3-4d42-a026-173aaccab9bb","type":"message","text":"Hi, is it possible to catch a specific type of exception in Julia? Been browsing around it seems that the workaround is with if-else, which seems pretty ugly, so was wondering if this has been supported in latest stable Julia version (1.5.3).","user":"UUT4VGTE2","ts":"1615180392.082700","team":"T68168MUP","blocks":[{"type":"rich_text","block_id":"aEC/4","elements":[{"type":"rich_text_section","elements":[{"type":"text","text":"Hi, is it possible to catch a specific type of exception in Julia? Been browsing around it seems that the workaround is with if-else, which seems pretty ugly, so was wondering if this has been supported in latest stable Julia version (1.5.3)."}]}]}],"thread_ts":"1615180392.082700","reply_count":1,"reply_users_count":1,"latest_reply":"1615180394.082800","reply_users":["B01J9QZ4SP8"],"subscribed":false},{"type":"message","subtype":"bot_message","text":"Can we cross post this publicly to <https://discourse.julialang.org|Discourse> for further visibility? React with :bridge: on the message above to approve. <https://github.com/JuliaCommunity/SlackBridge/blob/main/README.md#faq|More info here>","ts":"1615180394.082800","username":"HelpDeskBot","bot_id":"B01J9QZ4SP8","thread_ts":"1615180392.082700","parent_user_id":"UUT4VGTE2"}]